### DISCUSSION TITLE
Unable to connect to MariaDB 11 database

### DISCUSSION THREAD

**Discussion by nunesbeto** [getredash/redash#7010] [Q&A]
We've migrated our database from MariaDB 10.3 to MariaDB 11.3. After this we're unable to connect to the database.

Redash can't get database schema or run queries.

> Schema refresh failed.
> Please try again later.

We don't have any clue of what is causing the problem. There is no useful information in the logs.

Could anyone give some help with this?

**Comment by slackmodel** [comment]
If you are using docker to run redash, please upgrade the redash image to 25.1.0.

Please refer to the following URL: https://github.com/getredash/redash/releases/tag/v25.1.0
